examples/onboarding/config.json

```diff
@@ -0,0 +1,225 @@
+              "size": 4,
+              "mount": "/share/home"
+            }
+          }
+        }
+      }
+    }
+  },
+  "resources": {
+    "adnode": {
+      "type": "vm",
+      "public_ip": "true",
+      "password": "variables.win_password",
+      "vm_type": "Standard_D4s_v3",
+      "image": "variables.ad_image",
+      "subnet": "infra",
+      "tags": [
+        "ad_server"
+      ]
+    },
+    "frontend": {
+      "type": "vm",
+      "public_ip": "true",
+      "password": "variables.win_password",
+      "vm_type": "Standard_D4s_v3",
+      "image": "variables.nv_image",
+      "subnet": "infra",
+      "tags": [
+      ]
+    },
+    "license": {
+      "type": "vm",
+      "vm_type": "Standard_D4s_v3",
+      "public_ip": true,
+      "image": "variables.image",
+      "subnet": "infra",
+      "tags": [
+ 	"cndefault",
+        "netapp-client",
+        "disable-selinux",
+	"join_domain"
+      ]
+    },
+    "headnode": {
+      "type": "vm",
+      "vm_type": "Standard_D8s_v3",
+      "public_ip": true,
+      "image": "variables.hpc_image",
+      "subnet": "compute",
+      "tags": [
+ 	"cndefault",
+        "netapp-client",
+        "pbsserver",
+        "loginnode",
+        "localuser",
+        "disable-selinux",
+	"join_domain"
+      ]
+    },
+   "compute": {
+      "type": "vmss",
+      "vm_type": "variables.vm_type",
+      "instances": 2,
+      "image": "variables.hpc_image",
+      "subnet": "compute",
+      "tags": [
+        "netapp-client",
+        "pbsclient",
+        "cndefault",
+        "localuser",
+        "disable-selinux",
+        "join_domain"
+      ]
+    }
+  },
+  "install": [
+    {
+      "script": "setup_win_ad.sh",
+      "type": "local_script",
+      "args": [
+        "variables.resource_group",
+        "$(<hostlists/tags/ad_server)",
+        "variables.ad_domain",
+        "variables.ad_admin",
+        "variables.win_password"
+      ]
+    },
+    {
+      "script": "setup_win_ad_nfs.sh",
+      "type": "local_script",
+      "args": [
+        "variables.resource_group",
+        "$(<hostlists/tags/ad_server)",
+        "variables.ad_domain",
+        "variables.ad_admin",
+        "variables.win_password"
+      ]
+    },
+    {
+      "script": "disable-selinux.sh",
+      "tag": "disable-selinux",
+      "sudo": true
+    },
+    {
+      "script": "cndefault.sh",
+      "tag": "cndefault",
+      "sudo": true
+    },
+    {
+      "script": "auto_netappfiles_mount.sh",
+      "tag": "netapp-client",
+      "sudo": true
+    },
+    {
+      "script": "localuser.sh",
+      "args": [
+        "$(<hostlists/tags/nfsserver)"
+      ],
+      "tag": "localuser",
+      "sudo": true
+    },
+    {
+      "script": "pbsdownload.sh",
+      "tag": "loginnode",
+      "sudo": false
+    },
+    {
+      "script": "pbsserver.sh",
+      "copy": [
+        "pbspro_19.1.1.centos7/pbspro-server-19.1.1-0.x86_64.rpm"
+      ],
+      "tag": "pbsserver",
+      "sudo": false
```

Comment:
please change sudo to true for all pbs install scripts

Reply:
done